packages/xo-web/src/common/editable/index.js

```diff
@@ -58,6 +63,7 @@ class Hover extends Component {
   }
 }
 
+// use `data-*` to add params to the handler
```

Comment:
`handler` or `onChange` function?

And you should probably more explicit as to how this values are passed.